types/double-metaphone/double-metaphone-tests.ts

```diff
@@ -0,0 +1,3 @@
+import doubleMetaphone = require('double-metaphone');
+
+const x: string = doubleMetaphone('Test');
```

Comment:
this is obviously wrong, AFAIK:
https://github.com/words/double-metaphone/blob/master/index.js#L1017
see my comment about return type, thx!